### ISSUE TITLE
KDE 6.0.2 x64 could not strap in package plasma-wayland-session error

### ISSUE THREAD

**Issue by francisuk1989** [archlinux/archinstall#2418]
I am having issue installing KDE 6.0.2 with archinstall 2.7.2 via x64

it gives an error with **plasma-wayland-session**, it also mentions about could not strap in package however my internet works fine and when i try to re-sync nothing happens,  I have also looked on arch packages but theres no such package as **plasma-wayland-session**
`https://archlinux.org/packages/?sort=&repo=Core&repo=Core-Testing&repo=Extra&repo=Extra-Testing&repo=Gnome-Unstable&repo=KDE-Unstable&repo=Multilib&repo=Multilib-Testing&repo=Testing&q=plasma-wayland-session&maintainer=&flagged=`

I have tried keyring that was suggested from https://github.com/archlinux/archinstall/issues/907

`pacman -Sy archlinux-keyring`

But nothing.

Screenshot
`https://snipboard.io/R8QDbd.jpg`

Edit: 3
if i go to https://archlinux.org/packages/extra/x86_64/plasma-wayland-session/ it redirects me to plasma-workspace

Edit 4
When using archinstall and then selecting profile Server, Once server is done installing, I used
```
pacman -Syy
pacman -S plasma-meta (or for a minimal installation plasma-desktop)
```
once is installed, I just issued

`dbus-run-session startplasma-wayland`

Everything seems to be working great at this point.

**Comment by Torxed** [comment]
Fixed in https://github.com/archlinux/archinstall/pull/2386

**Comment by francisuk1989** [comment]
> Fixed in #2386

Just wonted to say, thanks very much! :+1:
